tests/test_basic.py

```diff
@@ -1029,6 +1029,18 @@ def test_jsonify_prettyprint():
         assert rv.data == pretty_response
 
 
+def test_jsonify_mimetype():
+    app = flask.Flask(__name__)
+    app.config.update({"JSONIFY_MIMETYPE": 'application/vnd.api+json'})
```

Comment:
I am following the conventions used in similar tests. For example: https://github.com/mitsuhiko/flask/pull/1728/files#diff-9e9ddedef65dec3da86063a067839e65R1021